ci_microshift.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+sudo yum install -y make golang
+
+./shellcheck.sh
+./microshift.sh
+
+# Run createdisk script
+# Set the zstd compression level to 10 to have faster
+# compression.
+export CRC_ZSTD_EXTRA_FLAGS="-10 --long"
+./createdisk.sh crc-tmp-install-data
+
+git clone https://github.com/code-ready/crc.git
```

Comment:
Perhaps add comment why we run with latest of crc.

Reply:
*grin* `/code-ready/`?